tapdb/sqlc/migrations/000052_rfq_forwards.up.sql

```diff
@@ -1,8 +1,8 @@
 CREATE TABLE IF NOT EXISTS rfq_forwards (
     id INTEGER PRIMARY KEY,
 
-    -- settled_at is the unix timestamp when the forward settled.
```

Comment:
Are the following commits meant to be fixups? If so some context on the fixup commit itself is needed, as many of the changes are not self explanatory (much of the previous diff is reverted)